sdk/search/Azure.Search.Documents/src/Azure.Search.Documents.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyTitle>Microsoft Azure.Search.Documents client library</AssemblyTitle>
+    <Version>1.0.0-preview.2</Version>
+    <Description>
+      This is the Azure Cognitive Search client library for developing .NET
+      applications with rich search experiences.  It enables you to query your
+      indexes, update documents in your indexes, and manage indexes, synonym
+      maps, indexers, data sources, or other service-level resources.
+    </Description>
+    <PackageTags>Azure Cognitive Search;Azure Search Documents;Azure Search;Search;Cognitive;Search Engine;Azure</PackageTags>
```

Comment:
Tip: FYI, this is a space-delimited field of single-world tags. Seems the service also parses out semicolons, but you have "Search" repated a bunch on the [package page](https://www.nuget.org/packages/azure.search).

Reply:
There's no way to group?  I'm going to leave this as we haven't changed but @AlexGhiondea may want to look into this across packages because Storage did the same.